docs/ldml/tr35.md

```diff
@@ -830,6 +830,16 @@ The BCP 47 form for keys and types is the canonical form, and recommended. Other
         <tr><td>"uksystem"</td>
             <td>UK System of measurement: feet, pints, etc.; pints are 20oz</td></tr>
 
+<tr><td colspan="4"><b>A <a name="MeasurementUnitPreferenceOverride" id="MeasurementUnitPreferenceOverride" href="#MeasurementUnitPreferenceOverride">Measurement Unit Preference Override</a> defines an override for measurement unit preference. The valid values are those <i>name</i> attribute values in the <i>type</i> elements of key name="mu" in bcp47/<a href="https://github.com/unicode-org/cldr/tree/main/common/bcp47/measure.xml" target="_blank">measure.xml</a></b>.</td></tr>
+<tr><td rowspan="3">"mu"</td>
+    <td rowspan="3">Measurement unit override</td>
+            <td>"celsius"</td>
+            <td>Celsius as temparature unit</td></tr>
```

Comment:
English spelling is `temperature` (not `temparature`).  Also for the other examples.